pkg/markers/crossplane.go

```diff
@@ -0,0 +1,19 @@
+package markers
+
+import "fmt"
+
+// CrossplaneOptions represents the Crossplane marker options that terrajet
+// would need to interact
+type CrossplaneOptions struct {
+	ReferenceToType string
+}
+
+func (o CrossplaneOptions) String() string {
```

Comment:
Similar comment with `Build()` vs `String()`. Having `String()` print the latest `//`-prefixed string looks easier to know which one users should interact with, but no strong opinion either way.

Reply:
`comments.Comment` is an object that you could build with a text (e.g. description) and some options. `String()` returns its string representation where all options are represented as markers. `Build()` on the other hand builds it as a comment to print. Still, I would agree for a simplification if we didn't have a use case where we also need to parse input text for markers, where using `String()` makes more sense, please see https://github.com/crossplane-contrib/terrajet/pull/50#discussion_r706215430